core/src/main/java/org/mule/construct/AbstractPipeline.java

```diff
@@ -91,6 +90,24 @@ protected MessageProcessor createPipeline() throws MuleException
         return builder.build();
     }
 
+    protected ProcessingStrategy createDefaultProcessingStrategy()
```

Comment:
this method is a fallback for creating the processing strategy you'll use if the user hasn't either set something on the flow, nor the configuration, nor the system property. Will add javadoc to better explain